gravitee-apim-console-webui/src/management/api/proxy/apis.proxy.route.html

```diff
@@ -77,7 +77,7 @@
           permission
           permission-only="['api-definition-r']"
           ui-sref-active="aui-nav-selected"
-          ui-sref="management.apis.detail.proxy.failover"
+          ui-sref="management.apis.detail.proxy.ng-failover"
```

Comment:
If you chnage the ui-sref I would remove the old controller and template.

If you keep the old controller and template I would let the old ui-sref. Then after validation from Antoine I would remove everything and do this change. 

You can do whatever you want both solutions are ok for me even if I prefer the second one personally. Wdyt ?